.github/workflows/assign-urgency-to-issue.yml

```diff
@@ -0,0 +1,304 @@
+            echo "in_project=false" >> "$GITHUB_OUTPUT"
+            exit 0
+          fi
+
+          echo "✅ Issue is in target project (item: $PROJECT_ITEM_ID)"
+
+          # Extract current urgency value from the specific project item we found
+          CURRENT_URGENCY=$(echo "$RESP" | jq -r --argjson pn "$PROJECT_NUMBER" --arg fieldName "$URGENCY_FIELD_NAME" \
+            '.data.repository.issue.projectItems.nodes[] | select(.project.number==$pn) | .fieldValues.nodes[] | select(.field.name==$fieldName) | .name // empty')
+
+          if [[ -n "$CURRENT_URGENCY" ]]; then
+            echo "📌 Current urgency: $CURRENT_URGENCY"
+          else
+            echo "📌 Current urgency: (not set)"
+          fi
+
+          # Store all labels for next step
+          LABELS_JSON=$(echo "$RESP" | jq -c '.data.repository.issue.labels.nodes | map(.name)')
+
+          echo "in_project=true" >> "$GITHUB_OUTPUT"
+          echo "issue_node_id=$ISSUE_NODE_ID" >> "$GITHUB_OUTPUT"
+          echo "project_v2_id=$TARGET_PROJECT_NODE_ID" >> "$GITHUB_OUTPUT"
+          echo "project_item_id=$PROJECT_ITEM_ID" >> "$GITHUB_OUTPUT"
+          echo "current_urgency=$CURRENT_URGENCY" >> "$GITHUB_OUTPUT"
+          echo "labels=$LABELS_JSON" >> "$GITHUB_OUTPUT"
+
+      - name: Check Labels
+        id: check_labels
+        if: steps.check_issue.outputs.in_project == 'true'
+        run: |
+          set -euo pipefail
+
+          LABELS='${{ steps.check_issue.outputs.labels }}'
+
+          # Extract and validate severity & likelihood using jq directly
+          severity=$(echo "$LABELS" | jq -r '.[] | select(startswith("severity/")) | sub("severity/"; "") | select(. == "low" or . == "mid" or . == "high" or . == "critical")' | head -n1)
+          likelihood=$(echo "$LABELS" | jq -r '.[] | select(startswith("likelihood/")) | sub("likelihood/"; "") | select(. == "low" or . == "mid" or . == "high")' | head -n1)
+
+          if [[ -z "$severity" ]]; then
+            echo "⚠️ No valid severity label found (expected: severity/low|mid|high|critical)"
+          else
+            echo "🏷️ Severity: $severity"
+          fi
+
+          if [[ -z "$likelihood" ]]; then
+            echo "⚠️ No valid likelihood label found (expected: likelihood/low|mid|high)"
+          else
+            echo "🏷️ Likelihood: $likelihood"
+          fi
+
+          echo "severity=$severity" >> "$GITHUB_OUTPUT"
+          echo "likelihood=$likelihood" >> "$GITHUB_OUTPUT"
+
+      - name: Calculate urgency
+        id: calculate_urgency
+        if: steps.check_issue.outputs.in_project == 'true'
+        run: |
+          set -euo pipefail
+          SEVERITY='${{ steps.check_labels.outputs.severity }}'
+          LIKELIHOOD='${{ steps.check_labels.outputs.likelihood }}'
+
+          if [[ -z "$SEVERITY" || -z "$LIKELIHOOD" ]]; then
+            echo "⚠️ Missing severity or likelihood; defaulting urgency to low"
+            URGENCY="low"
```

Comment:
❓ Wouldn't it be safer to mark it as high instead? If one of these is missing from a bug I'd argue the rootcausing isn't complete and we don't know the impact it the bug has. For all we know it could be really critical?